### ISSUE TITLE
unknown token "??" in my asr model predictions

### ISSUE THREAD

**Issue by marziye-A** [NVIDIA/NeMo#6651]
hello
while trying to finetune my asr model with another language, there is "??" token in my predictions while it isn't in my list of tokens.
what is it? does any one know?

**Comment by titu1994** [comment]
When finetuning in a new language you should build a new tokenizer (see the CTC fine-tune tutorial). Make sure all text in train set is covered and dev and test do not have additional tokens outside the train set (other special characters etc)

?? Is the UNK token for Sentencepiece

**Comment by marziye-A** [comment]
thank you very much! it really helped me a lot.

**Comment by lakshmi-speak** [comment]
When fine-tuning a model in the same language, in the interest of leveraging the trained weights for the decoder, I would like to retain the original tokenizer.  Is there a way to strip punctuation characters, not part of the tokenizer during the training? I am setting `cfg.train_ds.normalize_transcripts = True` but still see UNK token in the target and predictions. Or should this be handled ahead of time?
